torch/_dynamo/debug_utils.py

```diff
@@ -762,6 +762,8 @@ def backend_accuracy_fails(gm, example_inputs, compiler_fn, only_fwd=False):
 
 backend_aot_accuracy_fails = functools.partial(backend_accuracy_fails, only_fwd=True)
 
+MINIFIER_SPAWNED = False
```

Comment:
Please add comment here, explaining why this exists, how it is used, and when we would be able to remove it